package.json

```diff
@@ -7,7 +7,7 @@
     "test": "npm run lint && npm run unit-test",
     "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
     "lint:docs": "markdownlint \"**/*.md\"",
-    "lint:js": "eslint ./ --report-unused-disable-directives",
+    "lint:js": "eslint --cache ./ --report-unused-disable-directives",
```

Comment:
Probably want to keep all the options either before or after the positional argument `./`.